charmcraft/logsetup.py

```diff
@@ -82,6 +82,11 @@ def ended_ok(self):
         """Cleanup after successful execution."""
         os.unlink(self._log_filepath)
 
+    def ended_interrupt(self):
+        """Clean up on keyboard interrupt."""
+        _logger.error("Exiting on keyboard interrupt.")
```

Comment:
this doesn't feel like a great user error. What about just "Interrupted"
And I would log a trace back but not show a trace back to the user. But I think that is already true?

Reply:
my apologies for merging as it stood, i'll be addressing your review comments in a followup :-(

as it stands it _doesn't_ log a trace. The logfile is getting deleted anyway so I didn't think it was worth it.
On the other hand, if `--verbose` was requested, we _could_ log the traceback. Sounds reasonable to me.